README.md

```diff
@@ -64,35 +52,59 @@ Example:
 resolution -mc eth,btc,DODGE,unknown -d brad.zil
 ```
 
-## Note
+### JavaScript/NodeJs
 
-When resolution hits an error it returns the error code instead of throwing. So if you see something like RECORD_NOT_FOUND you know exactly that record was not found for this query.
+Install dependency
+```
+npm install @unstoppabledomains/resolution --save
+```
 
-# Contributor guide
+Example:
+```
+const { Resolution } = require('@unstoppabledomains/resolution');
+
+const resolution = new Resolution();
+resolution.address('resolver.crypto', "ETH")
+    .then(console.log)
+    .catch(console.error);
+```
 
-Paste that in a macOS Terminal or Linux shell prompt.
---
+For more information take a look [documentation](https://docs.unstoppabledomains.com/#tag/resolution_library)
 
-1) Install NVM
+## Note
 
-```bash
- curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
-```
+When resolution hits an error it returns the error code instead of throwing. So if you see something like RECORD_NOT_FOUND you know exactly that record was not found for this query.
 
-2) Install concrete version of node.js
+## Development
 
-```bash
- nvm install 12.12.0
-```
+Use next commands for setting up development environment. (**macOS Terminal** or **Linux shell**).
 
-3) Install ```yarn```
+1. Install NVM
+    ```bash
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+    ```
 
-```bash
- npm install -g yarn
-```
+2. Install concrete version of node.js
+    ```bash
+    nvm install 12.12.0
+    ```
 
-4) Download dependencies 
+3. Install ```yarn```
+    ```bash
+    npm install -g yarn
+    ```
+4. Clone repo
+    ```
+    git clone https://github.com/unstoppabledomains/resolution.git
+    cd resolution
+    ```
 
-```bash
-yarn install
-```
+5. Install dependencies 
+    ```bash
+    yarn install
+    ```
+
+# References
+
+1. [API Referrence](https://unstoppabledomains.github.io/resolution/)
+2. [Documentation](https://docs.unstoppabledomains.com/)
```

Comment:
I think "API Referrence" link should be on top. Generally, we should optimize readme for user not for contributor, so all the contributor info should be after user info.

Please remove the "Documentation" link. It is deprecated to host any resolution documentation code there.

I am not sure where "CDN" link has gone. I am also not sure it is useful for anyone. Consider removing or keeping.